ui/src/app/views/new-dataset-modal.tsx

```diff
@@ -205,11 +241,25 @@ class NewDataSetModal extends React.Component<Props, State> {
             {seePreview ? 'Hide Preview' : 'See Code Preview'}
           </Button>
           {seePreview && <React.Fragment>
-            {queries.length === 0 && <SpinnerOverlay />}
-            <TextArea disabled={true} onChange={() => {}} style={{marginTop: '1rem'}}
+            {Array.from(queries.values())
```

Comment:
There's a lot of code here that duplicates export-data-set-modal.  Can we refactor this?

Reply:
There's actually a ticket for that in the backlog. https://precisionmedicineinitiative.atlassian.net/secure/RapidBoard.jspa?rapidView=30&projectKey=RW&view=planning&selectedIssue=RW-2674&epics=visible&selectedEpic=RW-2014